### DISCUSSION TITLE
Can you produce a minimal executable example project based on the existing architecture of this repo?

### DISCUSSION THREAD

**Discussion by kjxbyz** [AppFlowy-IO/AppFlowy#3262] [Ideas]
Can you produce a minimal executable example project based on the existing architecture of this repo?

Other developers can quickly create a high-quality and reliable project based on a minimal example.

The license of this repo is AGPL-3.0 license. If I do this minimal example, will I be bound by this license?

**Comment by annieappflowy** [comment]
If you make your example open source, it's totally fine.
Have you tried out https://github.com/AppFlowy-IO/appflowy-editor
It offers an example

**Reply by kjxbyz** [reply]
I envision an app architecture with rust as the backend, flutter as the frontend, and grpc for communication.
